### PR TITLE
duplicate pending article/page

### PR DESCRIPTION
**Pull request by penk85**
WPC-616: Fixed duplicate article/page mutation to include pending.

### CODE REVIEW

File: packages/api/src/graphql/mutation.private.ts [hunk 1]
```diff
@@ -774,11 +774,14 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
 
         if (!article) throw new NotFound('article', id)
 
-        const articleRevision = Object.assign(article.published ?? article.draft, {
-          slug: '',
-          publishedAt: undefined,
-          updatedAt: undefined
-        })
+        const articleRevision = Object.assign(
+          article.published ?? article.pending ?? article.draft,
```
Comment (roosnic1) [inline packages/api/src/graphql/mutation.private.ts:778]: I actually would suggest you change the order. First check if there is a `draft` if not check for `pending` and only then take the `published`. I think this is more intuitively for the users.

**Review by roosnic1 — COMMENTED** [review]
Left a comment what do you think?

File: packages/api/src/graphql/mutation.private.ts (hunk 1 above)
Comment (penk85) [reply packages/api/src/graphql/mutation.private.ts:778]: @roosnic1 good point

**Review by roosnic1 — APPROVED** [review]

**Review by camilleory — APPROVED** [review]
👍

**Pull request merged**